lib/susedistribution.pm

```diff
@@ -198,6 +198,7 @@ sub become_root {
     my ($self) = @_;
 
     $self->script_sudo('bash', 0);
+    wait_still_screen;
```

Comment:
we can't wait in sudo because bash only returns when you exit the root-shell. My concern is that wait_still_screen will just timeout as the cursor blinks, doesn't it?